api/testdata/test.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: my-test
   namespace: eksa-packages
 spec:
-  packageName: Test
+  packageName: test
```

Comment:
Did you test this? Why do I feel like this has to match line 9 exactly (ie case-sensitively)?

Reply:
Yeh, I tested, the comparison is case insensitive line 9 should technically be lower case too

Reply:
🤷🏻 ok If either works, I don't see why we're changing things, but if it works, I can't object.

/lgtm